mobius/examples/mirror_tiling.rs

```diff
@@ -0,0 +1,114 @@
+use std::{f64::consts::PI, io::Error};
+
+use abstraction::Monoid;
+use mobius::{
+    algorithms::{OrbitIFS, OrbitTile},
+    cline_arc::ClineArc,
+    geometry::LineSegment,
+    hyperbolic_tilings::{get_fundamental_region, reflection_group},
+    isogonal::Isogonal,
+    motifs::candy_corn,
+    quantized_hash::QuantizedHash,
+    rendering::Style,
+    rotation, scale,
+    svg_plot::{render_views, style_geometry, style_motifs, union, View},
+    transformable::{ClineArcTile, Transformable},
+    translation, Complex,
+};
+
+pub fn better_candy_corners() -> Result<(), Error> {
+    let (conj, r_conj, e2_conj) = reflection_group(3, 7).unwrap();
+
+    let complex = e2_conj * Complex::Zero;
+    let dist_to_edge = 0.5 * (complex).real();
+    let (corn, styles) = candy_corn();
+    let (fundamental_domain, (_, _, vertex)) = get_fundamental_region(3, 7).unwrap();
```

Comment:
If there's a _quick_ way to simplify the hyperbolic tiling interface (maybe by passing in `p, q` and getting back a struct of everything you might want?), consider that.